uiowa/uiowa
# [CCOM] Analytics split

Resolves #7091 





# How to test



```
git pull && ddev composer install && ddev blt ds --site=community.medicine.uiowa.edu && ddev drush @medicinecommunity.local uli
```

Check that the piwik pro module is on, and that you can adjust the settings as an admin (the settings should be savable and ignored to be configurable by each site, but not having a valid id for it, this is difficult to actually test)

Check that the campus-wide gtm is not added on page load

```
ddev blt ds --site=default && ddev drush @default.local uli
```
Check that the piwik pro module is not on, and that campus-wide gtm is added on page load

Assuming the above works as expected, the config value is properly working on setting (or not) the campus-wide gtm container.

```
ddev drush @default.local cget uiowa_core.settings
```
See that `uiowa_core:campus_gtm` is `1`

```
ddev drush @default.local en ccom_core && ddev drush @default.local cim -y && ddev drush @default.local cget uiowa_core.settings
```
Check that enabling the module and config import goes smoothly, and that `uiowa_core:campus_gtm` is now `0`

```
ddev blt di && ddev drush @default.local cget uiowa_core.settings
```
See that `uiowa_core:campus_gtm` is `1` on a new install.
